samples/snippets/detect_intent_audio.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DialogFlow API Detect Intent Python sample with audio file.
+
+Examples:
+  python detect_intent_audio.py -h
+  python detect_intent_audio.py --agent AGENT \
+  --session-id SESSION_ID --audio-file-path resources/hello.wav
+"""
+
+import argparse
+import uuid
+
+from google.cloud.dialogflowcx_v3beta1.services.sessions import SessionsClient
+from google.cloud.dialogflowcx_v3beta1.types import audio_config
+from google.cloud.dialogflowcx_v3beta1.types import session
+
+
+def run_sample():
+    # TODO(developer): Replace these values when running the function
+    project_id = "YOUR-PROJECT-ID"
+    location_id = "YOUR-LOCATION-ID"
+    agent_id = (
+        "YOUR-AGENT-ID"  # https://cloud.google.com/dialogflow/cx/docs/concept/agent
+    )
+    agent = f"projects/{project_id}/locations/{location_id}/agents/{agent_id}"
+    session_id = str(
+        uuid.uuid4()
+    )  # https://cloud.google.com/dialogflow/cx/docs/concept/session
+    audio_file_path = "YOUR-AUDIO-FILE-PATH"
+    language_code = "en-us"
```

Comment:
Is this always `en-us`? If it is, can we move below into the sample? If not, can add a link to a list of supported options?